chart/prometheus-msteams/Chart.yaml

```diff
@@ -2,4 +2,4 @@ apiVersion: v1
 appVersion: "1.1.4"
 description: A Helm chart for Kubernetes
 name: prometheus-msteams
-version: 0.3.0
+version: 0.3.1
```

Comment:
@Flowkap thank you for this contribution. I am wondering whether this is a patch version API change but rather a major version API change? I assume `apiVersion: apps/v1` is not compatible with Kubernetes versions `<1.16`, is it?

Reply:
It is I believe since 1.12? Therefore and as the helm chart doesn't change it's own config API I chose lesser version.

Reply:
Thanks, I should have clicked your link first ;). I think this looks good then. @bzon please provide a lgtm, then we can merge.